app/src/main/java/net/aiscope/gdd_app/ui/metadata/MetadataPresenter.kt

```diff
@@ -74,4 +74,13 @@ class MetadataPresenter @Inject constructor(
         val current = repository.current()
         view.captureImage(current.nextImageName())
     }
+
+    suspend fun editImage(index: Int) {
+        val current = repository.current()
```

Comment:
Not a huge deal-breaker, but I think that instead of passing the index around, which is not very reliable (technically, the index in the adapter might be different than the one from the domain model), what if we would pass the mask `File`s down to the adapter along with the image `File`s, which then would make it straightforward to display the mask over the image in the metadata screen (but that part in a future PR maybe). We would kill two birds with one stone.

Reply:
Alright, updated to use the actual image/mask file